config/db_router.py

```diff
@@ -0,0 +1,63 @@
+from django.conf import settings
+from django.db import DEFAULT_DB_ALIAS
+
+
+class SecondaryDatabaseRouter:
```

Comment:
This name and docstring are pretty confusing because they imply that this is the router for the secondary db, but it is in fact the router for _all_ DBs.

Reply:
I will change it. Though, it is possible to have multiple routers as well.

Reply:
Updated